src/Infrastructure.EntityFramework/Repositories/CipherRepository.cs

```diff
@@ -107,7 +107,16 @@ private async Task<CipherDetails> CreateAsyncReturnCipher(CipherDetails cipher)
                 null;
             var entity = Mapper.Map<Cipher>((Core.Entities.Cipher)cipher);
             await dbContext.AddAsync(entity);
-            await dbContext.UserBumpAccountRevisionDateByCipherIdAsync(cipher.Id, cipher.OrganizationId.GetValueOrDefault());
```

Comment:
I think we should probably also make this method not allow a nullable (`Guid?`) organization Id so that this mistake is harder to make in the future. This would require us to make this be non-nullable as well but that is in line with what the SQL Server implementation is so I think it's fine. 
https://github.com/bitwarden/server/blob/522df6e91a937e02fba018a0276bb1561fd40e3d/src/Infrastructure.EntityFramework/Repositories/Queries/UserBumpAccountRevisionDateByCipherIdQuery.cs#L10

We should also get rid of this constructor that looks like it's only used in one test so that is fine with me. 
https://github.com/bitwarden/server/blob/522df6e91a937e02fba018a0276bb1561fd40e3d/src/Infrastructure.EntityFramework/Repositories/Queries/UserBumpAccountRevisionDateByCipherIdQuery.cs#L12-L16